<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui_tab_workout</class>
 <widget class="QWidget" name="ui_tab_workout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Available Exercises</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="exercise_name_filter"/>
        </item>
        <item>
         <widget class="QPushButton" name="toggle_only_low_progress">
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>‚≠ê</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="left_area_scroll">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>357</width>
             <height>433</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout"/>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Workout</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="exercise_type">
            <property name="currentText">
             <string>Workout</string>
            </property>
            <item>
             <property name="text">
              <string>Workout</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Journey</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Incidental</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="exercise_time">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="displayFormat">
             <string>yyyy/MM/dd HH:mm</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>üíñ</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Revert</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Exercises</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="exercise_info">
            <property name="font">
             <font>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="text">
             <string>test üòá test</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="right_area_scroll">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>494</width>
             <height>389</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>ui_tab_workout</receiver>
   <slot>event_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>released()</signal>
   <receiver>ui_tab_workout</receiver>
   <slot>event_revert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>843</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>released()</signal>
   <receiver>ui_tab_workout</receiver>
   <slot>event_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>887</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>857</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggle_only_low_progress</sender>
   <signal>released()</signal>
   <receiver>ui_tab_workout</receiver>
   <slot>event_toggle_low_progress_only()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>event_save()</slot>
  <slot>event_revert()</slot>
  <slot>event_close()</slot>
  <slot>event_toggle_low_progress_only()</slot>
 </slots>
</ui>
